admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss

```diff
@@ -1,96 +1,90 @@
 #right-sidebar {
-  background: white;
-  float: right;
   position: fixed;
-  display: block;
   top: 136px;
-  bottom: 0px;
-  z-index: 1000;
-  overflow: auto;
   right: -3em;
+  bottom: 0;
+  z-index: 1000;
+  display: block;
+  float: right;
   padding: 0 3em 0 0;
-  box-shadow: 0 0 12px rgba(0,0,0,0.8);
+  overflow: auto;
+  background: #fff;
+  box-shadow: 0 0 12px rgba(0, 0, 0, 0.8);
 }
 
-/*right sidebar is untested */
+/* right sidebar is untested */
 .sidebar.sidebar-right.sidebar-xs-show {
   right: 0;
-  -webkit-transform: translate(0,0);
-  -ms-transform: translate(0,0);
-  -o-transform: translate(0,0);
-  -moz-transform: translate(0,0);
-  transform: translate(0,0);
+  transform: translate(0, 0);
+}
+
+
+@media (max-width: 992px) {
```

Comment:
I think we have predefined breakpoints somewhere. Why don't you use those?

Reply:
You're right, forgot I swapped this file into scss one

It's fixed